KarhooUISDK/Screens/Rides/RidesList/RideCell/RideCellViewModel.swift

```diff
@@ -50,6 +50,8 @@ final class RideCellViewModel {
         if let freeCancellationMinutes = trip.serviceAgreements?.serviceCancellation.minutes,
            freeCancellationMinutes > 0 {
             freeCancellationMessage = String(format: UITexts.Quotes.freeCancellation, "\(freeCancellationMinutes)")
+        } else if trip.serviceAgreements?.serviceCancellation.type == "BeforeDriverEnRoute" {
```

Comment:
Could we have the types as an Enum?

Reply:
Ah, that would require some refactoring in the KarhooSDK.